modules/compliance-supported-profiles.adoc

```diff
@@ -64,13 +87,17 @@ The Compliance Operator provides the following compliance profiles:
 |0.1.39+
 |link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
 |`x86_64`
+ `ppc64le`
```

Comment:
We don't need to add `s390x` or `ppc64le` support for `rhcos4-moderate`, just `ocp4-moderate` and `ocp4-moderate-node`.

Reply:
Done! I don't know how you knew that, let's discuss later.